subprojects/base-services/src/main/java/org/gradle/internal/progress/BuildOperationDescriptor.java

```diff
@@ -21,21 +21,27 @@
 /**
  * Meta-data about a build operation.
  */
-public class BuildOperationDetails {
-    private final BuildOperationExecutor.Operation parent;
+public class BuildOperationDescriptor {
+    private final Long id;
```

Comment:
Afaik it's only used for progress events, which should be referring to build operations anyway. They shouldn't have their own id.